lib/wallet/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -2363,6 +2375,65 @@ balanceTransactionSpec = describe "balanceTransaction" $ do
     it "produces valid transactions or fails"
         $ property prop_balanceTransactionValid
 
+    describe "bootstrap witnesses" $ do
+        -- Used in 'estimateTxSize' in turn used by coin-selection
+        let coinSelectionEstimatedSize :: Natural -> Natural
+            coinSelectionEstimatedSize
+                = fromIntegral . sizeOf_BootstrapWitnesses . fromIntegral
+
+        let measuredWitSize :: IsCardanoEra era => Cardano.Tx era -> Natural
+            measuredWitSize (Cardano.Tx body wits) = fromIntegral $
+                  serializedSize (Cardano.Tx body wits)
+                - serializedSize (Cardano.Tx body [])
+
+        let evaluateMinimumFeeSize
+                :: forall era. WriteTx.IsRecentEra era
+                => Cardano.Tx era
+                -> Natural
+            evaluateMinimumFeeSize (Cardano.Tx body wits) = fromIntegral
+                $ WriteTx.unCoin
+                $ WriteTx.evaluateMinimumFee
+                    (WriteTx.recentEra @era)
+                    pp
+                    (WriteTx.fromCardanoTx (Cardano.Tx body []))
+                    (KeyWitnessCount 0 (fromIntegral $ length wits))
+              where
+                -- Dummy PParams to ensure a Coin-delta corresponds to a
+                -- size-delta.
+                pp = case WriteTx.recentEra @era of
+                    RecentEraBabbage -> setField @"_minfeeA" 1 def
+                    RecentEraAlonzo  -> setField @"_minfeeA" 1 def
+
+        let evaluateMinimumFeeDerivedWitSize (Cardano.Tx body wits) =
+                  evaluateMinimumFeeSize (Cardano.Tx body wits)
+                - evaluateMinimumFeeSize (Cardano.Tx body [])
```

Comment:
```suggestion
        let evaluateMinimumFeeDerivedWitSize (Cardano.Tx body wits)
                = evaluateMinimumFeeSize (Cardano.Tx body wits)
                - evaluateMinimumFeeSize (Cardano.Tx body [])
```

Reply:
TBH, I like the visual prominence of the `-` which currently exists

Reply:
> TBH, I like the visual prominence of the `-` which currently exists

I only mentioned this because _strictly_ speaking it's inconsistent with our coding standards:
https://input-output-hk.github.io/adrestia/code/Coding-Standards#avoid-variable-length-indentation

However, I don't feel terribly strongly about it in this particular case.

Reply:
> Other code uses a pattern where everything is aligned at multiple of the indent level (which is consistent with the standard):

Ok, fair enough 👍